src/components/dock/dock-button/dock-button.tsx

```diff
@@ -153,4 +172,25 @@ export class DockButton {
             );
         }
     }
+
+    private setMenuFocus() {
+        if (!this.isOpen) {
+            return;
+        }
+
+        setTimeout(() => {
+            console.log('id', this.portalId);
+            const list: HTMLElement = document.querySelector(
+                `#${this.portalId} limel-menu-surface limel-list`
+            );
+            console.log('LIST', list);
+
+            const firstItem: HTMLElement =
```

Comment:
If we have a reference to the custom component element here, we can just do this:

```typescript
if (element.delegatesFocus) {
    element.focus();
    return;
}
```

Reply:
Yes I confirm that `delegatesFocus` works as we want. I will push the fix shortly.

Reply:
I added `shadow: { delegatesFocus: true }` to the menu component in the example.